developer-docs-site/docs/nodes/validator-node/operator/connect-to-aptos-network.md

```diff
@@ -268,6 +268,41 @@ After your validator node joined the validator set, you can verify the correctne
     
     You should expect the active value for your `StakePool` to keep increasing. It is updated at every epoch.
 
+## Seeing your stake pool information
+
+To see the details of your stake pool, run the below CLI command with the `get-stake-pool` option by providing the `--owner-address` and `--url` fields. This example is for Premainnet. For other networks, use the appropriate REST URL for the `--url` field. See [Aptos Blockchain Deployments](/nodes/aptos-deployments) for `--url` field values for other networks. 
+
+The below command is for an example owner address `0x0756c80f0597fc221fe043d5388949b34151a4efe5753965bbfb0ed7d0be08ea` and it will display the following information on the owner's stake pool on the premainnet:
+
+- Status of the stake: pending active, active, pending-inactive, inactive.
+- Total stake.
+- Consensus key.
+- Addresses (pool, operator, voter). 
+- Unlocked commission.
+- Fullnode network address.
+- Epoch information.
+- Lockup expiration information.
+
+:::tip Use CLI 0.3.8 or higher
+Make sure you use the CLI version 0.3.8 or higher. See [Installing Aptos CLI](/cli-tools/aptos-cli-tool/install-aptos-cli.md).
+- Type `aptos --help` to see the CLI version.
+- Type `aptos node get-stake-pool --help` for more on the command option for the below example.
+:::
+
+```bash
+aptos node get-stake-pool \
+  --owner-address 0x0756c80f0597fc221fe043d5388949b34151a4efe5753965bbfb0ed7d0be08ea \
+  --url https://premainnet.aptosdev.com
+```
+
+## Checking your validator performance
+
+To see your validator performance in the current and past epochs and rewards earned, run the below command. The output will show the validator's performance in block proposals and in governance voting and governance proposals. Default values are used in the below command. Type `aptos node analyze-validator-performance --help` to see default values used.
+
+```bash
+aptos node analyze-validator-performance --analyze-mode All \
+  --url https://premainnet.aptosdev.com
+```
```

Comment:
Example output:

{
  "Result": {
    "current_epoch_successful_proposals": 56,
    "current_epoch_failed_proposals": 0,
    "previous_epoch_rewards": [
      "12312716242",
      "12272043711",
      "12312912674",
      "12313011054",
      "12313109435",
      "12180092056",
      "12313305136",
      "12313403519",
      "12313501903",
      "12313600288"
    ],
    "epoch_info": {
      "epoch": 68,
      "epoch_interval": 3600000000,
      "last_epoch_start_time": {
        "unix_time": 1665074662417326,
        "utc_time": "2022-10-06T16:44:22.417326Z",
        "local_time": "Thu Oct  6 16:44:22 2022"
      },
      "next_epoch_start_time": {
        "unix_time": 1665078262417326,
        "utc_time": "2022-10-06T17:44:22.417326Z",
        "local_time": "Thu Oct  6 17:44:22 2022"
      }
    }
  }
}

Reply:
I am suspecting this output is not for `-All` option. The output only as epoch info so I am guessing `--analyze-mode=detailed-epoch-table`? I think?